release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2852,3 +2852,19 @@ The Insights Operator can now gather `Datahubs` resources from SAP clusters. Thi
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-27"]
+=== RHBA-2021:1427 - {product-title} 4.6.27 bug fix update
+
+Issued: 2021-05-04
+
+{product-title} release 4.6.27 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:1427[RHBA-2021:1427] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:1428[RHBA-2021:1428] advisory.
```

Comment:
Could golf it to:
```suggestion
{product-title} release 4.6.27 is now available. The bug fixes that are included in this update are documented in the link:https://access.redhat.com/errata/RHBA-2021:1427[RHBA-2021:1427] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:1428[RHBA-2021:1428] advisory.
```

Grain of salt and all of that, though--it's all fine boilerplate. :smile:

Reply:
Your suggestion is closer to how it used to be - was changed to this to be _ever-so-slightly_ more accurate :joy: